cli/test/test_utility.py

```
@@ -0,0 +1,80 @@
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        {
+            "os_name": "posix",
+            "home_path": "/home/testuser",
+            "local_appdata": None,
+            "expected_path": "/home/testuser/.config/potpie/potpie.pid",
+        }
+    ],
+)
+def test_pid_file_creation(monkeypatch, test_params):
```

Ruff (0.8.2)

Redefinition of unused `test_pid_file_creation` from line 7

(F811)